### PR TITLE
[hi] Add content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md

### PR DESCRIPTION
**Pull request by vedant-kakde**
Localizes :

- `content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md`

### CODE REVIEW

**Comment by vedant-kakde** [comment]
This PR is in continuation of hi localization effort: #29353

/language hi

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+यदि आपके पास kubectl के लिए एक उपनाम है, तो आप उस उपनाम के साथ काम करने के लिए शेल कम्पलीशन को बढ़ा सकते हैं:
+
+```zsh
+echo 'alias k=kubectl' >>~/.zshrc
+echo 'complete -F __start_kubectl k' >>~/.zshrc
+```
+
+अपने शेल को पुनः लोड करने के बाद, kubectl ऑटो-कम्पलीशन कार्य करना चाहिए.
```
Comment (kaiwalyakoparkar) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md:22]:
```diff
- अपने शेल को पुनः लोड करने के बाद, kubectl ऑटो-कम्पलीशन कार्य करना चाहिए.
+ अपने शेल को पुनः शुरू  करने के बाद, kubectl ऑटो-कम्पलीशन कार्य करना चाहिए.
```

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
This sounds more appropriate. What do you think about this?

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: "zsh ऑटो-कम्पलीशन"
+description: "zsh ऑटो-कम्पलीशन के लिए कुछ वैकल्पिक कॉन्फ़िगरेशन|"
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md:3]:
```suggestion
description: "zsh ऑटो-कम्पलीशन के लिए कुछ वैकल्पिक कॉन्फ़िगरेशन।"
```

Comment (anubha-v-ardhan) [reply content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md:3]: Please note that `।` and `|` are not the same character and we should use `।` ideally.

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+headless: true
+---
+
+Zsh के लिए kubectl कम्पलीशन स्क्रिप्ट `kubectl completion zsh` कमांड के साथ उत्पन्न की जा सकती है। आपके शेल में कम्पलीशन स्क्रिप्ट को सोर्स करने से kubectl ऑटो-कम्पलीशन सक्षम हो जाती है|
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md:7]:
```suggestion
Zsh के लिए kubectl कम्पलीशन स्क्रिप्ट `kubectl completion zsh` कमांड के साथ उत्पन्न की जा सकती है। आपके शेल में कम्पलीशन स्क्रिप्ट को सोर्स करने से kubectl ऑटो-कम्पलीशन सक्षम हो जाती है।
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md (hunk 1 above)
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md:22]:
```suggestion
अपने शेल को पुनः लोड करने के बाद, kubectl ऑटो-कम्पलीशन कार्य करना चाहिए।
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+यदि आपके पास kubectl के लिए एक उपनाम है, तो आप उस उपनाम के साथ काम करने के लिए शेल कम्पलीशन को बढ़ा सकते हैं:
+
+```zsh
+echo 'alias k=kubectl' >>~/.zshrc
+echo 'complete -F __start_kubectl k' >>~/.zshrc
+```
+
+अपने शेल को पुनः लोड करने के बाद, kubectl ऑटो-कम्पलीशन कार्य करना चाहिए.
+
+अगर आपको कोई त्रुटि मिलती है जैसे `complete:13: command not found: compdef`, फिर अपनी `~/.zshrc` फ़ाइल की शुरुआत में निम्नलिखित जोड़ें:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md:24]:
```suggestion
यदि आपको कोई त्रुटि मिलती है जैसे `complete:13: command not found: compdef`, तो अपनी `~/.zshrc` फ़ाइल की शुरुआत में निम्नलिखित जोड़ें:
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by vedant-kakde** [comment]
Done ✅

**Comment by anubha-v-ardhan** [comment]
/lgtm
/approve

Thank you @vedant-kakde :)

**Pull request merged**